docs/Diagnostics.md

```diff
@@ -0,0 +1,87 @@
+This document contains information about diagnostic tools and outputs used for debugging and diagnosing eBPF issues.
+
+--------------------
+
+- [WFP State](#wfp-state)
+- [bpftool](#bpftool)
+- [eBPF Diagnostic Traces](#ebpf-diagnostic-traces)
+  - [Trace Providers](#trace-providers)
+  - [Logman Trace Command](#logman-trace-command)
+  - [Decoding Traces](#decoding-traces)
+  - [Viewing Traces](#viewing-traces)
+
+--------------------
+
+## WFP State
```

Comment:
This section appears without any explanation of why WFP is relevant.  Seems like it should be under some subsection about the specific program types that WFP is relevant for.